### PR TITLE
Jkralik/fix/tls buffer conn

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: coap-gateway/service/publishedResources.go [hunk 1]
```diff
@@ -14,9 +14,11 @@ type publishedResources struct {
 }
 
 func (p *publishedResources) Add(href ...string) {
-	p.lock.Lock()
-	defer p.lock.Unlock()
-	p.resources = p.resources.Insert(href...)
+	/*
```
Comment (Danielius1922) [inline coap-gateway/service/publishedResources.go]: This probably doesn't belong in this PR.

Comment (jkralik) [reply coap-gateway/service/publishedResources.go]: fixed

**Comment by Danielius1922** [comment]
The patch is not needed in `Dockerfile.test` file?

**Comment by jkralik** [comment]
> The patch is not needed in `Dockerfile.test` file?

Good point. Fixed

**Review by Danielius1922 — APPROVED** [review]
